modules/auxiliary/scanner/http/emby_version.rb

```diff
@@ -0,0 +1,87 @@
+##
```

Comment:
I would move this to `emby_version_ssrf` since its using SSRF to determine the version.